### PR TITLE
Dsp9049/playing page update

### PR DESCRIPTION
**Pull request by dsp9049**

### CODE REVIEW

File: src/user_interface/configPopup.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    def __init__(self):
+        super().__init__()
+        stacked = QStackedLayout()
+        gray = QRect()
+        layout = QVBoxLayout()
+        self.setWindowFlags(QtCore.Qt.FramelessWindowHint)
+        self.setGeometry(100, 100, 500, 200)
+        layout.addLayout(IconsList())
+        layout.addLayout(Piano())
```
Comment (cha2251) [inline src/user_interface/configPopup.py:18]: Actually I lied, one more thing. This looks good on my desktop, but on my laptop's smaller screen it overlaps a few other things. Can you put a boarder on it so it looks a little cleaner?

**Review by cha2251 — CHANGES_REQUESTED** [review]

File: src/user_interface/configPopup.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+from PyQt5 import QtGui, QtCore
```
[the remark this answers is not in the document]
Comment (dsp9049) [reply src/user_interface/configPopup.py:1]: configuration_popup.py shouldn't exist

File: src/user_interface/configPopup.py (hunk 1 above)
Comment (dsp9049) [reply src/user_interface/configPopup.py:19]: I put the default windowed boarder around it, in the future I want to make it so it grays out the main window, which will prevent this popup from blending in the background too much

**Review by cha2251 — APPROVED** [review]
lgtm!

**Pull request merged**